### PR TITLE
New Locomotion bindings

### PR DESCRIPTION
**Pull request by JeroMiya**
Adds bindings for the locomotion interfaces (position and velocity), and added a binding to check the context status which was also missing.

### CODE REVIEW

File: ClientKit/ClientKit.cs [hunk 1]
```diff
@@ -80,6 +80,9 @@ public class ClientContext : IDisposable
             [DllImport(OSVRCoreDll, CallingConvention = CallingConvention.Cdecl)]
             public extern static Byte osvrClientGetStringParameter(SafeClientContextHandle ctx, string path, StringBuilder buf, UIntPtr len);
 
+            [DllImport(OSVRCoreDll, CallingConvention = CallingConvention.Cdecl)]
+            internal extern static Byte osvrClientCheckStatus(SafeClientContextHandle ctx);
```
Comment (rpavlik) [inline ClientKit/ClientKit.cs:84]: any particular reason why this is internal when the one above it isn't?

File: ClientKit/ClientKit.cs [hunk 2]
```diff
@@ -397,6 +400,22 @@ protected virtual void Dispose(bool disposing)
...
+            /// If this reports that the client context is not OK, there may not be a server
+            /// running, or you may just have to call osvrClientUpdate() a few times to
+            /// permit startup to finish. The return value of this call will not change from
+            /// failure to success without calling osvrClientUpdate().
+            /// </summary>
+            /// <returns>false if not yet fully connected/initialized, or if
+            /// some other error (null context) occurs.</returns>
+            public bool CheckStatus()
+            {
+                return osvrClientCheckStatus(this.m_context) == OSVR_RETURN_SUCCESS;
```
Comment (rpavlik) [inline ClientKit/ClientKit.cs:415]: Suppose this is a good reason :)

File: ExampleClients/Locamotion/Locamotion.cs [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+/// distributed under the License is distributed on an "AS IS" BASIS,
+/// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+/// See the License for the specific language governing permissions and
+/// limitations under the License.
+/// </copyright>
+
+using System;
+using OSVR.ClientKit;
+
+namespace Locamotion
```
Comment (rpavlik) [inline ExampleClients/Locamotion/Locamotion.cs]: Spelling - all should be `locomotion`

File: ClientKit/ClientKit.cs (hunk 1 above)
Comment (JeroMiya) [inline ClientKit/ClientKit.cs:84]: They will all be internal eventually. Didn't want to add new public P/Invokes that I'd have to `[Obsolete]` later.

**Pull request merged**